app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt

```diff
@@ -126,9 +146,26 @@ class ProfilePictureActivityTest {
 
   @Test
   fun testProfilePictureActivity_userImageIsDisplayed() {
-    launch<ProfilePictureActivity>(createProfilePictureActivityIntent(internalProfileId)).use {
-      onView(withId(R.id.profile_picture_image_view)).check(matches(isDisplayed()))
-    }
+    activityTestRule.launchActivity(createProfilePictureActivityIntent(internalProfileId))
+    testCoroutineDispatchers.runCurrent()
+    onView(withId(R.id.profile_picture_image_view)).check(matches(isDisplayed()))
+  }
+
+  @Test
+  fun testProfilePictureActivity_hasCorrectTitle() {
+    activityTestRule.launchActivity(createProfilePictureActivityIntent(internalProfileId))
+    testCoroutineDispatchers.runCurrent()
+    val profilePictureActivityTitle = context.getString(R.string.profile_picture_activity_title)
+    onView(withId(R.id.profile_picture_activity_toolbar))
+      .check(matches(hasDescendant(withText(profilePictureActivityTitle))))
+  }
+
+  @Test
+  fun testProfilePictureActivity_closesOnPressingNavigateUpButton() {
```

Comment:
Generally, it's clearer to structure the test name as: testAction_withCondition_andOtherAction_hasOutcome, e.g.:

```suggestion
  fun testProfilePictureActivity_pressNavigateUp_finishesActivity() {
```

Reply:
okay got it